statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -261,85 +280,263 @@ def var_loglike(resid, omega, nobs):
 
     .. math::
 
-        -\left(\frac{T}{2}\right)
-        \left(\ln\left|\Omega\right|-K\ln\left(2\pi\right)-K\right)
+       -\left(\frac{nobs}{2}\right)
+       \left(\ln\left|cov_sigma_mle\right|-neqs\ln\left(2\pi\right)-neqs\right)
     """
-    logdet = logdet_symm(np.asarray(omega))
-    neqs = len(omega)
+
+    logdet = logdet_symm(np.asarray(cov_resid_mle))
+    neqs = len(cov_resid_mle)
     part1 = - (nobs * neqs / 2) * np.log(2 * np.pi)
     part2 = - (nobs / 2) * (logdet + neqs)
     return part1 + part2
 
 def _reordered(self, order):
```

Comment:
also reordering the results requires that data is copied.
Is it possible to make it optional, or redesign?